### PR TITLE
feat: [IOAPPPEB-7] Handles only the user activation status

### PR DESCRIPTION
**Pull request by CrisTofani**
## Short description
This PR changes the logic to handle the showing or not of the optin flow to consider only the bpd activation status and not to loading period information.

### CODE REVIEW

File: ts/screens/wallet/WalletHomeScreen.tsx [hunk 1]
```diff
@@ -222,9 +218,7 @@ class WalletHomeScreen extends React.PureComponent<Props, State> {
 
     // To maintain retro compatibility, if the opt-in payment methods feature flag is turned off,
```
Comment (Vangaorth) [inline ts/screens/wallet/WalletHomeScreen.tsx:219]: Shouldn't this comment be removed/updated?

Comment (CrisTofani) [reply ts/screens/wallet/WalletHomeScreen.tsx:219]:
```suggestion
```

sure

File: ts/features/bonus/bpd/saga/orchestration/optInPaymentMethods/optInShouldShowChoiceHandler.ts [hunk 2]
```diff
@@ -33,13 +39,19 @@ export function* optInShouldShowChoiceHandler(): Generator<
   any
 > {
   // Load the information about the participation of the user to the bpd program
-  yield* put(bpdAllData.request());
+  yield* put(bpdLoadActivationStatus.request());
   const bpdAllDataResponse = yield* take<
```
Comment (Vangaorth) [inline ts/features/bonus/bpd/saga/orchestration/optInPaymentMethods/optInShouldShowChoiceHandler.ts:43]: Maybe this variable name should be changed?

Comment (CrisTofani) [reply ts/features/bonus/bpd/saga/orchestration/optInPaymentMethods/optInShouldShowChoiceHandler.ts]: fixed in [c0a7f02](https://github.com/pagopa/io-app/pull/4173/commits/c0a7f026ddaf56b69670b3d3b971c867f9d05bee)

**Review by Vangaorth — APPROVED** [review]
LGTM

**Pull request merged**